pkg/review/commit_review_status_test.go

```diff
@@ -0,0 +1,454 @@
+                       nodes{
+                         databaseId,
+                         number,
+                         reviewDecision
+                       },
+                       pageInfo{
+                         endCursor,
+                         hasNextPage,
+                         hasPreviousPage,
+                         startCursor
+                       },
+                       totalCount
+                     }
+                   }
+                 }
+               }
+             }
+           ",
+           "variables": {
+             "commitSha": "kof6p96lr6qvdu81qw49fhmoxrod9qmc2qak51nh",
+             "githubOrg": "test-org",
+             "pageCursor": "",
+             "repository":"test-repo"
+           }
+         }`,
+			},
+			want: []*PullRequest{},
+			responseBodies: []string{
+				`{
+           "data": {
+             "repository": {
+               "object": {
+                 "associatedPullRequests": {
+                   "nodes": [],
+                   "pageInfo": {
+                     "endCursor": "XQ",
+                     "hasNextPage": false,
+                     "hasPreviousPage": false,
+                     "startCursor": ""
+                   },
+                   "totalCount": 0
+                 }
+               }
+             }
+           }
+         }`,
+			},
+		},
+	}
+	for _, tc := range cases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+			gotRequestBodies := make([]string, len(tc.wantRequestBodies))
+			requestNumber := 0
+			fakeGitHub := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				expectedAuthHeader := fmt.Sprintf("Bearer %s", tc.token)
+				if r.Header.Get("Authorization") != expectedAuthHeader {
+					w.WriteHeader(500)
+					fmt.Fprintf(w, "missing or malformed authorization header")
+					return
+				}
+				bytes, _ := io.ReadAll(r.Body)
+				requestBody := string(bytes)
+				gotRequestBodies[requestNumber] = requestBody
+				fmt.Fprintf(w, tc.responseBodies[requestNumber])
+				requestNumber++
+			}))
+			src := oauth2.StaticTokenSource(
+				&oauth2.Token{AccessToken: tc.token},
+			)
+			ctx := context.Background()
+			httpClient := oauth2.NewClient(ctx, src)
+			client := githubv4.NewEnterpriseClient(fakeGitHub.URL, httpClient)
+			got, err := GetPullRequests(ctx, client, tc.githubOrg, tc.repository, tc.commitSha)
+			if diff := cmp.Diff(got, tc.want); diff != "" {
+				t.Errorf("GetPullRequests got unexpected result (-got,+want):\n%s", diff)
+			}
+			if diff := testutil.DiffErrString(err, tc.wantErr); diff != "" {
+				t.Errorf("Process(%+v) got unexpected err: %s", tc.name, diff)
+			}
+			for i := range tc.wantRequestBodies {
+				got := gotRequestBodies[i]
+				want := tc.wantRequestBodies[i]
+				if !equalExceptWhiteSpace(got, want) {
+					t.Errorf("Incorrect Request Body: got=%s, want=%s", got, want)
+				}
+			}
```

Comment:
The assumption seems to be both "got" and "want" will have the same size. Either check that, or normalize the whole slice of "got" and "want" and use `cmp.Diff` to compare, which will take care of the size comparison.

Reply:
done.